pkg/limits/ingest_limits.go

```diff
@@ -179,51 +171,46 @@ func NewIngestLimits(cfg Config, lims Limits, logger log.Logger, reg prometheus.
 }
 
 func (s *IngestLimits) Describe(descs chan<- *prometheus.Desc) {
-	descs <- tenantPartitionDesc
-	descs <- tenantRecordedStreamsDesc
-	descs <- tenantActiveStreamsDesc
+	descs <- partitionsDesc
+	descs <- tenantStreamsDesc
 }
 
 func (s *IngestLimits) Collect(m chan<- prometheus.Metric) {
 	cutoff := s.clock.Now().Add(-s.cfg.WindowSize).UnixNano()
-
-	var (
-		recorded            = make(map[string]int)
-		active              = make(map[string]int)
-		partitionsPerTenant = make(map[string]map[int32]struct{})
-	)
-
+	// active counts the number of active streams (within the window) per tenant.
+	active := make(map[string]int)
+	// expired counts the number of expired streams (outside the window) per tenant.
+	expired := make(map[string]int)
 	s.metadata.All(func(tenant string, partitionID int32, stream Stream) {
-		if assigned := s.partitionManager.Has(partitionID); !assigned {
```

Comment:
I think we should remove this, we want to know if a tenants streams are being over-counted.